be/db.go

```diff
@@ -12,9 +12,18 @@ import (
 )
 
 var (
-	mysqlAddress = flag.String("mysql_address", "server:dev_pass@tcp(cleanupdb:3306)/cleanapp", "MySQL address string")
+	mysqlPassword = flag.String("mysql_password", "secret", "MySQL password.")
+	mysqlHost = flag.String("mysql_host", "cleanupdb", "MySQL host.")
```

Comment:
Let's make the default value localhost. cleanappdb isn't a default one, at least for running on a localhost. And

Reply:
Done.